### ISSUE TITLE
App crashes on device with API 24

### ISSUE THREAD

**Issue by ntsiam** [ptzianos/hermes#2]
Trace:
```
2019-05-14 14:37:34.545 31335-32007/org.hermes I/IOTAConnector: a4e23fb0-7ec3-4d84-820b-9cf5dae771a9 -- Broadcasting of transactions with addresses: ...
2019-05-14 14:37:35.237 31335-32007/org.hermes E/AndroidRuntime: FATAL EXCEPTION: pool-1-thread-1
    Process: org.hermes, PID: 31335
    java.lang.IllegalAccessError: 429 
        at org.iota.jota.connection.HttpConnector.wrapCheckedException(HttpConnector.java:254)
        at org.iota.jota.connection.HttpConnector.attachToTangle(HttpConnector.java:327)
        at org.iota.jota.IotaAPICore.attachToTangle(IotaAPICore.java:622)
        at org.iota.jota.IotaAPI.sendTrytes(IotaAPI.java:428)
        at org.hermes.ledgers.IOTAConnector.sendData(IOTAConnector.kt:138)
        at org.hermes.LedgerService.broadcastData(LedgerService.kt:316)
        at org.hermes.LedgerService$broadcastData$1.invokeSuspend(LedgerService.kt)
        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
        at kotlinx.coroutines.DispatchedTask.run(Dispatched.kt:233)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:607)
        at java.lang.Thread.run(Thread.java:762)
```

**Comment by ptzianos** [comment]
A fix is available in commit 5bc6310. @ntsiam plz test

**Comment by ntsiam** [comment]
It doesn't seem to be crashing anymore, so I close the issue.
